pkg/kwokctl/runtime/compose/compose.go

```diff
@@ -62,8 +62,8 @@ type BuildComposeConfig struct {
 	SecretPort bool
 	QuietPull  bool
 
-	ApiserverPort  uint32
-	PrometheusPort uint32
+	KubeApiserverPort int
```

Comment:
same here, why did these change to int?

Reply:
There are more uint32 and int conversions back and forth, I'll change it to uint32 uniformly